documentation/src/main/asciidoc/topics/performance.adoc

```diff
@@ -131,6 +131,14 @@ threads to the number of concurrent client requests:
 include::config_examples/hotrod_worker_threads.xml[]
 ----
 
+= SSL provider
+
+{brandname} server will automatically use the native implementation of the OpenSSL libraries on supported platforms to
```

Comment:
https://github.com/infinispan/infinispan/pull/9176/files#diff-645d9d401f15e89e8572dab73e051a8fe2ce7c29283bdc2addf19a9a562defa0R293 seems to say otherwise

Reply:
I've amended the client `package-info.java` because it will default to openssl there too.